.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx pretty-quick
```

Comment:
I've played around with `husky` a bit and it worked nicely so far. Will be good to add `--staged` to check staged files as well!

```sh
npx pretty-quick --staged
```